docs/JestObjectAPI.md

```diff
@@ -647,7 +647,7 @@ Set the default timeout interval for tests and before/after hooks in millisecond
 
 _Note: The default timeout interval is 5 seconds if this method is not called._
 
-_Note: The method must be called after the test framework is installed in the environment and before the test runs. A good place to do this is in the `setupTestFrameworkScriptFile`._
+_Note: The method must be called after the test framework is installed in the environment and before the test runs. A good place to do this is in the `jest.config` with `setupFilesAfterEnv`._
```

Comment:
"good place to do this" refers to "the method must be called", which happens in the "setupFilesAfterEnv", not in the config, so I think the previous wording made more sense, but of course with the new config key
```suggestion
_Note: The method must be called after the test framework is installed in the environment and before the test runs. A good place to do this is in the `setupFilesAfterEnv`._
```

Also, maybe we should mention this as "if you want to set the timeout for all tests, ..." and also suggest doing it for just one test?

Reply:
I don't know Jest well enough yet, but the JesObjectAPI.md could be improved by clearly stating what can or cannot be applied in the setupFilesAfterEnv. Seems like you would either go with a test based change or globally in setupFilesAfterEnv but as a new comer i found it hard to break into the concept by just using the docs which should not be the case.

Reply:
I agree with your comment though and pushed a new commit reflecting this.